uwp/Add-Syncfusion-Controls.md

```diff
@@ -25,12 +25,12 @@ Syncfusion UI for UWP are added automatically to the Visual Studio Toolbox durin
 
 2. Find `SfTextBoxExt` by typing the text “SfTextBoxExt” in search box.
 
-![Search SfTextBoxExt](Add-Syncfusion-Controls_images/Add-Syncfusion-Controls_img1.jpeg)
+![Search SfTextBoxExt in UWP](add-syncfusion-controls_images/uwp-add-syncfusion-controls-search-sftextboxext.jpeg)
 
 
 3.Drag `SfTextBoxExt` and drop it in the designer.
 
-![Drag SfTextBoxExt in to Designer](Add-Syncfusion-Controls_images/Add-Syncfusion-Controls_img2.jpeg)
+![Drag SfTextBoxExt in to Designer in UWP](add-syncfusion-controls_images/uwp-add-syncfusion-controls-drag-sftextboxext.jpeg)
```

Comment:
Why you mentioned add-syncfusion-controls in all image name. 
We should use proper platform, control name and its feature related term in image name.
we can change the image name like as,
add-uwp-textbox-control-through-designer

Reply:
Changed the image name